src/components/suggest/ch-suggest.tsx

```diff
@@ -146,6 +151,12 @@ INDEX:
   itemSelectedHandler(event: CustomEvent<SuggestItemSelectedEvent>) {
```

Comment:
If the "ch-suggest" will re-emit the ch-suggest-item event, you should stop propagating that event, that is, add:
```js
event.stopPropagation();
```

Reply:
That makes sense. Thank you.